### ISSUE TITLE
PyramidFlow训练demo会开放吗

### ISSUE THREAD

**Issue by ChHanXiao** [gasharper/autoFlow#2]

**Comment by laiyingxin2** [comment]
同问

**Comment by gasharper** [comment]
Yes, we plan to release an improved training code in another repository in the future, which will allow for reproducing the paper results. This repository will remain user-friendly and easy to use. It is a collection of models (like timm) but does not include other stuff (like training code). Furthermore, we intend to maintain this repository with other SOTA normalization flow models to increase the influence of this methodology.

**Comment by weiliangwuw** [comment]
> Yes, we plan to release an improved training code in another repository in the future, which will allow for reproducing the paper results. While the _autoFlow_ repository will remain user-friendly and easy to use. It is a collection of models (like timm) but does not include other stuff (like training code). Furthermore, we intend to maintain this repository with other SOTA normalization flow models to increase the influence of this methodology.

具体什么时候开放呀

**Comment by gasharper** [comment]
在六月中旬的main conference前。
